zebrad/src/application.rs

```diff
@@ -45,6 +45,10 @@ pub struct ZebradApp {
 }
 
 impl ZebradApp {
+    fn is_tty() -> bool {
```

Comment:
Unix has 3 standard streams, input, output, and error. TTYs can also be attached to other file descriptors.
```suggestion
    /// Are standard output and standard error both connected to ttys?
    fn outputs_are_ttys() -> bool {
```